riscv-debug-spec.tex

```diff
@@ -619,6 +642,34 @@ \subsection{JTAG Connector}
 connector looks as it does in Table~\ref{tab:header}. The function of each pin
 is described in Table~\ref{tab:pinout}.
 
+%MAW: Why do we only have 'RESET', but no 'TRST(n)' signal?
+% See http://openocd.org/doc/html/Reset-Configuration.html
+% Both are useful.
+% One is explicitly ``test logic reset'', which would imply
+% resetting the JTAG state machine and all associated test logic, in this
+% case the Debug Transport Module, synchronizers, and perhaps even
+% the Debug Module itself. It basically does the same thing as getting
+% the JTAG state machine into TEST_LOGIC_RESET state, which is
+% why it is optional.
+%
+% The other is a convenience which allows the debugger to essentially
+% hit the system reset button, which is basically mutually exclusive from
+% what TRST resets.
```

Comment:
I agree that we could require it, but I don't know why we'd want to.

Earlier you made an argument based on the JTAG spec saying only TRST can reset the TAP (or at least that's what I understood). That doesn't sound right because clearly the spec says TRST is optional.

Does it make more sense to continue this discussion in e-mail? It's getting a bit confusing having to go through the pull request to see which thread has new comments.